code/game/objects/structures/stairs.dm

```diff
@@ -16,6 +16,20 @@
 	var/terminator_mode = STAIR_TERMINATOR_AUTOMATIC
 	var/turf/listeningTo
 
+
+/obj/structure/stairs/north
+	dir = NORTH
+
+/obj/structure/stairs/south
+	dir = SOUTH
+
+/obj/structure/stairs/east
+	dir = EAST
+
+/obj/structure/stairs/west
+	dir = WEST
```

Comment:
I don't think this is needed. Mappers can generate a pallet placeable objects per icon state or per direction, with SDMM at least.

Reply:
^

Reply:
We don't need this